# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

TurnActionGoal action_goal
TurnActionResult action_result
TurnActionFeedback action_feedback
